docs/content/en/blog/releases/v0.41.0.md

```diff
@@ -6,4 +6,68 @@ description: >
  Release v0.41.0
 ---
 
+## Changes since v0.40.1
 
+### Breaking Changes
+
+* Remove trigger paths ([#4037](https://github.com/pipe-cd/pipecd/pull/4037))
+
+All __applications configuration__ file should NOT use the `spec.triggerPaths` (marked as deprecated previously), but use `spec.trigger.onCommit.paths` instead.
+
+### New features
+
+From now, you can manage your project's deployment via PipeCD deployment metrics. Please refer to the [docs](/docs/user-guide/insights/).
+
+![image](/images/insights.png)
+
+* Introduce Insights feature ([#4071](https://github.com/pipe-cd/pipecd/pull/4071))
+* Support setting kubectlVersion via piped config ([#4081](https://github.com/pipe-cd/pipecd/pull/4081))
+* Add `spec.trigger.onCommit.ignores` to application config ([#4059](https://github.com/pipe-cd/pipecd/pull/4059))
+
+### Notable Changes
+
+* Add `--include-crds` option to include CRDS for Helm ([#4077](https://github.com/pipe-cd/pipecd/pull/4077))
+* Ensure newly created revision is ready to receive traffic before going to the next stage ([#4074](https://github.com/pipe-cd/pipecd/pull/4074))
+* Update web dependencies ([#4061](https://github.com/pipe-cd/pipecd/pull/4061))
+* Use helper "serviceAccountName" on create role binding ([#4054](https://github.com/pipe-cd/pipecd/pull/4054))
+
+### Internal Changes
+
+* Improve the health description for Cloud Run live state ([#4085](https://github.com/pipe-cd/pipecd/pull/4085))
+* Refactor the loadCloudRunManifest function ([#4089](https://github.com/pipe-cd/pipecd/pull/4089))
+* Enable to run lint/go on M1 ([#4088](https://github.com/pipe-cd/pipecd/pull/4088))
+* Implement lint/go command for local ([#4086](https://github.com/pipe-cd/pipecd/pull/4086))
+* Fix the wrong link for tools ([#4083](https://github.com/pipe-cd/pipecd/pull/4083))
+* Remove unused docs ([#4082](https://github.com/pipe-cd/pipecd/pull/4082))
+* Support Cloud Run application for planpreview ([#4079](https://github.com/pipe-cd/pipecd/pull/4079))
```

Comment:
I suppose we should add this to the new features category.

Reply:
Ooops, nice catch 🙏